scene/debugger/scene_debugger.cpp

```diff
@@ -99,6 +99,21 @@ void SceneDebugger::_handle_input(const Ref<InputEvent> &p_event, const Ref<Shor
 	}
 }
 
+void SceneDebugger::_handle_embed_input(const Ref<InputEvent> &p_event, const Dictionary &p_settings) {
+	Ref<InputEventKey> k = p_event;
+	Ref<Shortcut> p_shortcut = p_settings.get("editor/suspend_resume_embedded_project", Ref<Shortcut>());
+	if (p_shortcut.is_valid() && k.is_valid() && k->is_pressed() && !k->is_echo() && p_shortcut->matches_event(k)) {
```

Comment:
`k.is_valid() && k->is_pressed() && !k->is_echo()` can be done once before everything.

Reply:
Oh, that's true, nice. Made the changes. 

I just separated the is_echo() check in another if a bit later. Because I actually want the next_frame button to work with echo to quickly hold and skip several frames.